cloudinit/util.py

```diff
@@ -1802,6 +1801,24 @@ def time_rfc2822():
         ts = "??"
     return ts
 
+def boottime():
+    import ctypes
+
+    NULL_BYTES = b"\x00"
+
+    class timeval(ctypes.Structure):
+        _fields_ = [
+            ("tv_sec", ctypes.c_int64),
+            ("tv_usec", ctypes.c_int64)
+        ]
+    libc = ctypes.CDLL('/lib/libc.so.7')
+    size = ctypes.c_size_t()
+    size.value = ctypes.sizeof(timeval)
+    buf = timeval
+    if libc.sysctlbyname(b"kern.boottime" + NULL_BYTES, ctypes.byref(buf),
+                      ctypes.byref(size), None, 0) != -1:
```

Comment:
indent the ctypes.byref under the opening parenthesis.

Reply:
aye, I don't have python indentation correctly set up for this project in emacs